### DISCUSSION TITLE
Adding advanced FAQ section on documentation

### DISCUSSION THREAD

**Discussion by manjitapandey** [hotosm/field-tm#1825] [General]
Users are mostly confused about few things on FMTM functionalities. Such questions might be like

- How to download and load MB tiles
- Can the xls form be edited after project creation.
- Is it possible to map not digitised features?
- Can we lock multiple tasks for mapping or validation purpose? .....
We can discuss and add more of such questions on thread and update the FAQ sections of FMTM.

cc: @ramyaragupathy , @ivangayton , @spwoodcock

**Comment by spwoodcock** [comment]
This is on my todo list to address some of these points in an FAQ 😃

It would be really appreciated if others could support more with the doc writing though 🙏🙏

**Comment by manjitapandey** [comment]
I can update the doc. Can you provide me some additional questions which can be added, that's in your mind? Or we can work on it together.

**Comment by spwoodcock** [comment]
There should definitely be info about the custom ODK Collect!

- why it's needed (a more seamless user experience we can't provide with the default Collect. We are working with the devs to get this functionality into official Collect)
- how to install it (currently it might require uninstalling the version installed, prior to installing the custom one, due to app version conflicts)
- how it works (data is sent from FMTM, filling out some forms in ODK Collect automatically. This prevents the user having to open the map to navigate to the feature in ODK, and instead we can provide more complex functionality in FMTM, like actual navigation route generation and directions to locate the feature)

The other questions you mentioned are good too!

**Comment by spwoodcock** [comment]
I made a start on this!

Feel free to update / contribute whatever is needed 😄 

https://docs.fmtm.dev/about/faq/#advanced-use-cases
https://github.com/hotosm/fmtm/blob/development/docs/about/faq.md#advanced-use-cases

Closing this!
